### ISSUE TITLE
 `TypeError: extractEmail is not a function`

### ISSUE THREAD

**Issue by yarekc** [gajus/extract-email-address#5]
test.mjs

```
import extractEmail from 'extract-email-address';
extractEmail('extracts email from anywhere within the input gajus@gajus.com');
```

**node test.msj**

`TypeError: extractEmail is not a function`

I also tried with require()

**Comment by hbrks** [comment]
Any update yet? i face the same issue

**Comment by tmokmss** [comment]
For me, require() did not work as well, but the below resolved the isssue. I'm not totally sure why but jfyi:

1. create a file `foo.d.ts` somewhere in your project

```ts
declare module 'extract-email-address' {
  function extractEmail(str: string): { email: string }[];
  export = extractEmail;
}
```

2. Set `"esModuleInterop": true` in tsconfig.json
3. Then the below code works:

```ts
import extractEmail from 'extract-email-address';
extractEmail('bar@example.com');
``` 

edit) It seems [email-addresses](https://www.npmjs.com/package/email-addresses) can be a better option.

**Comment by gajus** [comment]
Fixed in v3.0.1
